src/client.ts

```diff
@@ -137,6 +137,7 @@ export class StreamChat<
   secret?: string;
   setUserPromise: ConnectAPIResponse<ChannelType, CommandType, UserType> | null;
   state: ClientState<UserType>;
+  testIPOverride?: string;
```

Comment:
Any name suggestions?

Reply:
@AnatolyRugalev is this field only for testing purposes? 
if yes I'd suggest removing it from client code completely and just stub the `connection._buildUrl` function in the test env

Reply:
Yes, this is only for testing. Good idea, will try that

Reply:
I got it working with basic suffixing of `userAgent` string. I tried to make mock work but I cant override `_buildUrl` of the connection because the `wsConnection` property is null while client is not connected. And when client is already connected changing URL is useless.